parse.py

```diff
@@ -1,45 +1,93 @@
+#!/usr/bin/python3
+import re
+import os
+import json
+import datetime
 import collections
 import itertools
-import json
-import os
-import re
-
-from dateutil import parser
 
-dir = os.getcwd() + "/Takeout/YouTube/"
-watch_history = dir + "history/watch-history.html"
-search_history = dir + "history/search-history.html"
-comment_history = dir + "my-comments/my-comments.html"
-like_history = dir + "playlists/likes.json"
+missing=[]
+dir = os.path.join(os.getcwd(),"Takeout/YouTube/")
+if not os.path.exists(dir):
+	missing.append(dir)
+found=False
+for path in ("Verlauf/Wiedergabeverlauf.html","history/watch-history.html"):	#translations
+	watch_history = os.path.join(dir,path)
+	if os.path.exists(watch_history):
+		found=True
+		break
+if not found:
+	missing.append(watch_history)
+found=False
```

Comment:
You should do it like this. Way more cleaner.
Also you should probably move the translations to their own file so that they can be re-used elsewhere  and be added later on.

```
for path in ("Verlauf/Wiedergabeverlauf.html","history/watch-history.html"):	#translations
	watch_history = os.path.join(dir,path)
	if os.path.exists(watch_history):
		break
else:
	missing.append(watch_history)
```